portfolio/src/main/webapp/script.js

```diff
@@ -47,18 +47,34 @@ function getComments() {
 function createListElement(text) {
   const liElement = document.createElement('li');
   liElement.className = 'comment';
+
   const nameElement = document.createElement('span');
-  nameElement.innerText = text.name;
+  nameElement.innerText = text.name + " ";
+
   const dateElement = document.createElement('span');
   const date = new Date(text.timestamp);
   const dateTime = " - " + date.toDateString() + " " + date.getHours() + ":" + date.getMinutes();
   dateElement.innerText = dateTime;
+
   const textElement = document.createElement('p');
   textElement.innerText = text.text;
 
+  const sentimentScoreElement = document.createElement('img');
+  const score = Math.round(text.sentimentScore * 100) / 100
+  if (score > 0.2) {
+      sentimentScoreElement.src = "/images/smile.png";
+  } else if (score < 0.2){
+      sentimentScoreElement.src = "/images/sad.png";
+  }else{
+      sentimentScoreElement.src = "/images/neutral.png";
+  }
```

Comment:
0.2 is a bit of a "magic number" right now, meaning it appears directly in the code but I have no idea what it means. I suggest adding a comment explaining why 0.2 in particular. (What are the range of values that the sentiment score could have? Is 0.2 a recommended number, or did you just pick it arbitrarily? By the way, picking it arbitrarily is totally fine, but you should note that in a comment so future editors aren't afraid to change it arbitrarily. :)  )

For extra credit, you could create a const named "SMILE_THRESHOLD" or something, set it to 0.2, and use that instead of the raw number everywhere else in your code. That way, if you change it later, you can rest assured that it's changed everywhere at once.